tt_metal/impl/device/device.cpp

```diff
@@ -1149,7 +1161,7 @@ void Device::update_workers_build_settings(std::vector<std::vector<std::tuple<tt
                     compile_args[22] = 0; // 22: test_results_addr (disabled)
                     compile_args[23] = 0; // 23: test_results_size (disabled)
                     compile_args[24] = 0; // 24: timeout_cycles
-                    compile_args[25] = 0xF; // 25: output_depacketize_mask
+                    compile_args[25] = 0x3; // 25: output_depacketize_mask
```

Comment:
Why does output_depacketize / cb mode is 0x3 now? All outputs from the vc_eth_tunneler need to be output_depacketize[i] = true so it was 0xF before

Reply:
This is the code path that only has 1-to-2 mux/demuxes, so changed this to match what shows up in the new code which is 0x3